change/@ni-nimble-components-907c9819-88b3-432c-b476-6104362e75bc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@mollykreis I disabled automerge as Trevor's bubble-up is actively going on (Angular 13 just merged and Angular 14 is coming immediately after). I'd recommend letting 14 go through and then continuing merge